workers_launch_template.tf

```diff
@@ -219,8 +219,9 @@ resource "aws_autoscaling_group" "workers_launch_template" {
       lookup(
         var.worker_groups_launch_template[count.index],
         "tags",
-        local.workers_group_defaults["tags"]
-      )
+        []
```

Comment:
this will produce empty tag which will be concat in line 193, so this whole list should be also compacted (in line 193)